python/lib/cloudutils/serviceConfig.py

```diff
@@ -23,6 +23,29 @@
 import os
 import shutil
 
+# exit() error constants
+Unknown = 0
+CentOS6 = 1
+CentOS7 = 2
+Ubuntu = 3
+RHEL6 = 4
+RHEL7 = 5
+
+#=================== DISTRIBUTION DETECTION =================
+if os.path.exists("/etc/redhat-release"):
+    version = file("/etc/redhat-release").readline()
+    if version.find("Red Hat Enterprise Linux Server release 6") != -1:
+        distro = RHEL6
+    elif version.find("Red Hat Enterprise Linux Server 7") != -1:
+        distro = RHEL7
+    elif version.find("CentOS release 6") != -1:
```

Comment:
We can detect/check for `/etc/centos-release` for CentOS distros?

Reply:
we can do that but i think in redhat binary copys they will be always from compatible things like /etc/redhat-release
what do you think?